cms/models.py

```diff
@@ -787,6 +789,18 @@ class Meta:
         blank=True,
         help_text="A short description indicating how long it takes to complete (e.g. '4 weeks')",
     )
+    FORMAT_CHOICES = [
+        (FORMAT_ONLINE, FORMAT_ONLINE),
+        (FORMAT_OTHER, FORMAT_OTHER),
+    ]
+    format = models.CharField(
+        max_length=20,
+        choices=FORMAT_CHOICES,
+        default=FORMAT_ONLINE,
+        null=True,
+        blank=True,
```

Comment:
I think we should not keep this null or blank. For past records, we are already giving a default value.